src/commands/run-tests.yml

```diff
@@ -1,105 +1,113 @@
 description: >
-  Run all tests in a MATLAB project and generate test artifacts. MATLAB includes any files in your
-  project that have a Test label. If your pipeline does not leverage a MATLAB project, then MATLAB
-  includes all tests in the the root of your repository including its subfolders.
+  Run MATLAB and Simulink tests and generate artifacts. By default, MATLAB includes any files in
+  your project that have a Test label. If your pipeline does not use a MATLAB project, or if it
+  uses a MATLAB release before R2019a, then MATLAB includes all tests in the root of your
+  repository and in any of its subfolders.
```

Comment:
I think it's important to add this sentence here:

The command fails if any of the included tests fails.